src/components/SVGViewer/SVGViewer.tsx

```diff
@@ -336,6 +338,16 @@ export class SVGViewer extends Component<SvgViewerProps, SvgViewerState> {
     });
   };
 
+  shouldDisableOptimization = () => {
+    // Bacause willChange optimization has a calculation cost
+    // we don't want to apply it for a bigger embedded into svg <image />
+    // tags since it slows down the performance instead of providing
+    // a boost. Images are already heavily optimizaide by browsers
```

Comment:
```suggestion
    // a boost. Images are already heavily optimized by browsers
```

Reply:
That was one more typo :P

Reply:
I even now noticed a grammar error in those lines of code but I'll let it be 😝